### ISSUE TITLE
Feature request: Allow voting in read-only rooms

### ISSUE THREAD

**Issue by lucymk** [sampaiodiego/rocket.chat.app-poll#46]
@sampaiodiego I haven't looked into the underlying stuff, maybe this is nigh on impossible - or maybe it should be working and there's a bug! 

Let me know what you think 😄

**Comment by sampaiodiego** [comment]
this is an interesting use case. for v2 I think the only issue is with app user's permissions, since its him who create and edit the message to show the votes, that might be the problem with https://github.com/sampaiodiego/rocket.chat.app-poll/issues/33 as well.

need a proper test and if that's the case just need to document it somewhere (and maybe give the user an alert about it) ;)

**Comment by vblg** [comment]
The problem is most likely due to Poll user rights. I made it admin and was able to create a vote in a read-only room. Sorry for my English.

**Comment by awunder** [comment]
it definitely is the app role permission.. give it the permission to post in read-only and you are good to go.. no need to give it admin

**Comment by sampaiodiego** [comment]
thanks @awunder for taking a look .. I didn't have time to proper look into this.

Even if I can't fix these errors, I need to add some error handling, show somewhere the error message if an error occurred

**Comment by awunder** [comment]
yeah, some user feedback would be great.. i "figured out" why it wont post the poll via issue search and finding this issue :)

**Comment by andys448** [comment]
Hi All! 
Any news?
Still not available create poll if room set is read-only
